recipe-server/normandy/settings.py

```diff
@@ -380,6 +388,9 @@ class Development(Base):
     SWAGGER_SETTINGS = Base.SWAGGER_SETTINGS
     SWAGGER_SETTINGS['VALIDATOR_URL'] = None
 
+    # URL that hosts Experimenter experiment definitions
+    EXPERIMENTER_HOST = values.URLValue('https://localhost')
```

Comment:
I can't imagine this is an accurate url for a local version of Experimenter. It should be optional.

We should also either include docs about how to setup Experimenter or add Experimenter to docker-compose to automatically be set up.